sycl/test-e2e/README.md

```diff
@@ -106,15 +159,15 @@ separated from comma-separated list of target devices with colon. Example:
 -DSYCL_TEST_E2E_TARGETS="opencl:cpu;ext_oneapi_level_zero:gpu;ext_oneapi_cuda:gpu;ext_oneapi_hip:gpu;ext_intel_esimd_emulator:gpu"
 ```
 
-***SYCL_BE*** - SYCL backend to be used for testing. Supported values are:
+***SYCL_BE*** - Deprecated. `SYCL backend to be used for testing. Supported values are:
  - **opencl** - for OpenCL backend;
  - **ext_oneapi_cuda** - for CUDA backend;
  - **ext_oneapi_hip** - for HIP backend;
  - **ext_oneapi_level_zero** - Level Zero backend;
  - **ext_intel_esimd_emulator** - ESIMD emulator backend;
 
 
-***SYCL_TARGET_DEVICES*** - comma separated list of target devices for testing.
+***SYCL_TARGET_DEVICES*** - Deprecated. Comma separated list of target devices for testing.
```

Comment:
Please, remove these instructions from the documentation. We must document supported options/environment variables.